Transformed the Conclusion article to a script, fixes #162 (#163)
InnerSourceCommons/InnerSourceLearningPath: contributor/06-conclusion-script.md

## Script

J:: Thanks for watching the Contributor segment of the InnerSource Commons Learning Path. With this section you've learned about the Contributor role - the life blood of InnerSource projects: External to the owners of a component's team, they bring additional valuable input to the project.

> I: You've learned about how to become an contributor by learning about how to find opportunities to contribute and the mindset and habits needed to find or create such opportunities.

Reviewer: ```suggestion
I: You've learned about how to become an contributor by looking for opportunities to contribute as well as the mindset and habits needed to find or create such opportunities.
```

Reviewer: To remove the duplication of "learn"

We've also discussed the ethos of the role and suggested some behaviour that is likely to lead to successful contributions.

J: Given the right mindset, habits and ethos there are still a few details that might keep you from successfully contributing - hence, we've discussed these nuts and bolts in greater detail.

I: Last but not least, convincing your teammates and your organization at various levels can be hard, thus we've discussed the benefits of contribution from various perspectives to make this process easier for you.   

> J: We hope you've enjoyed watching and reading the articles and videos and hope you could take away some interesting new insights for your journey towards InnerSource and being a good contributor. 

Reviewer: ```suggestion
J: We hope you've enjoyed watching the videos and reading the articles. We hope you could take away some interesting new insights for your journey towards InnerSource and being a good contributor. 
```

I: If you haven't done so already, we would like to invite you to also learn more about the other aspects of InnerSource in our InnerSource learning path: http://innersourcecommons.org/learningpath/ 

-> Show link overlay.

Be welcome to additionally check out the InnerSource group [InnerSource Commons](http://innersourcecommons.org) online! 

> Feel free to join the discussion and share your experiences and lessons learnt in your organization.

Reviewer: ```suggestion
Feel free to join the discussion and share your experiences and lessons learnt.
```

Reviewer: To remove the duplicate "your"
